R/qualification-ddi.R

```diff
@@ -457,7 +454,10 @@ getDDISection <- function(dataframe, metadata, sectionID, idPrefix, captionSuffi
       id = tableID,
       sectionId = sectionID,
       table = ddiTableList[[pkParameter]],
-      tableCaption = paste("Summary table for", metadata$title, "-", pkParameter, "Ratio"),
+      tableCaption = paste(
+        "Summary table for", metadata$title, "-", pkParameter, "Ratio.",
+        "(\u03b4 =", metadata$guestDelta[[pkParameter]], "in Guest *et al.* formula)"
+      ),
```

Comment:
some blanks are missing: 
* after **"Summary table for"** (unless the title starts with a blank)
* before and after **"Ratio."**
* after **"(\u03b4 ="**
* before **"in Guest *et al.* formula)"**

Reply:
yes, it does. closing the comment :)